test/api/v1/organizations/organizationsController.spec.js

```diff
@@ -0,0 +1,103 @@
+const assert = require('assert');
+const proxyquire =  require('proxyquire').noCallThru();
+const sinon = require('sinon');
+
+let queryResolve;
+let organizationsController;
+let queryStub;
+
+describe('organizationsController', () => {
+  beforeEach(() => {
+    queryResolve = {
+      query: () => Promise.resolve()
+    };
+
+    queryStub = sinon.stub(queryResolve, 'query').returns(Promise.resolve());
+
+    organizationsController = proxyquire('api/v1/organizations/organizationsController', {
+      '../../../services/neo4jHelpers': queryResolve
+    });
+  });
+
+  afterEach(() => {
+    queryStub.restore();
+  });
+
+  describe('#deleteOrganization', () => {
+    it('should return a rejected promise when called without a valid UUID', (done) => {
+      organizationsController.deleteOrganization().catch(() => done());
+    });
+
+    it('should return a rejected promise when called without a valid UUID', (done) => {
+      organizationsController.deleteOrganization('invalid-id').catch(() => done());
+    });
+
+    it('should pass the ID in the second argument to .query()', () => {
+      const guid = '110ec58a-a0f2-4ac4-8393-c866d813b8d1';
+      const queryPromise = organizationsController.deleteOrganization(guid);
+      const secondArgPassedToQuery = queryResolve.query.getCall(0).args[1];
+
+      assert.deepEqual(secondArgPassedToQuery, {id: guid});
+    });
+  });
+
+  describe('#getOrganization', () => {
+    it('should return a rejected promise when called without a valid UUID', (done) => {
+      organizationsController.getOrganization().catch(() => done());
+    });
+
+    it('should return a rejected promise when called without a valid UUID', (done) => {
+      organizationsController.getOrganization('invalid-id').catch(() => done());
+    });
+
+    it('should pass the ID in the second argument to .query()', () => {
+      const guid = '110ec58a-a0f2-4ac4-8391-c866d813b8d1';
+      const queryPromise = organizationsController.getOrganization(guid);
+      const secondArgPassedToQuery = queryResolve.query.getCall(0).args[1];
+
+      assert.deepEqual(secondArgPassedToQuery, {id: guid});
+      queryStub.restore();
+    });
+  });
+
+  // TODO: Not sure this should be organization... Maybe it should be organizations?
+  describe('#getOrganizations', () => {
```

Comment:
OK. I am going to re-check through the testing aspect of things. I'll probably need some help later on and have some questions. I just saw it passed the git commit tests and thought it was ok.